doc/requirements.txt

```diff
@@ -4,3 +4,4 @@ devlib>=0.0.3
 psutil>=4.4.2
 wrapt>=1.10.8
 ipython>=4.2.0
+matplotlib>=1.4.2
```

Comment:
Why isn't this in the previous commit?

Reply:
Will fixup that bit, thanks for pointing it out.